pkg/storage/pvc/pvc.go

```diff
@@ -116,13 +117,36 @@ func (d *driver) checkPVC(cr *imageregistryv1.Config, claim *corev1.PersistentVo
 		}
 	}
 
+	// Check what access modes are available.
+
+	// We allow using RWO PV backend, but it has some limitations:
+	// 1. Image registry rollout strategy must be set to Recreate (default is RollingUpdate).
+	// 2. It's not possible to use more than 1 replica of the image registry.
+
+	// RWX backends are accepted with no additional conditions.
+	rwoModeEnabled := false
```

Comment:
this validation looks reasonable to me, thanks for the update.  

For other reviewers, if RWX is **anywhere** in the list the volume will be mounted so multiple pods can use it.  If RWO is the **only** thing in the list then it will be mounted so only a single pod can use it.  If both exist the volume is mounted to support both so it falls into the first use case.

Reply:
thanks :) everything you said is correct